apps/fluent-tester/src/TestComponents/CornerRadius/CornerRadiusTest.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import { HOMEPAGE_CORNERRADIUS_TESTPAGE } from './consts';
+import { Test, TestSection, PlatformStatus } from '../Test';
+import { View } from 'react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import { stackStyle } from '../Common/styles';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import { Text } from '@fluentui/react-native';
+import { useFluentTheme } from '@fluentui-react-native/framework';
+import { getCurrentAppearance } from '@fluentui-react-native/theming-utils';
+
+interface CornerRadiusTestComponentProps {
+  name: string;
+  cornerRadius: number;
+}
+
+const CornerRadiusTestComponent: React.FunctionComponent<CornerRadiusTestComponentProps> = (props: CornerRadiusTestComponentProps) => {
+  const theme = useFluentTheme();
+  const isLightMode = getCurrentAppearance(theme.host.appearance, 'light') === 'light';
+
+  const outerBoxProps = React.useMemo(
+    () => ({
+      height: 60,
+      margin: 16,
+      backgroundColor: isLightMode ? globalTokens.color.grey80 : globalTokens.color.grey10,
+      borderRadius: props.cornerRadius,
+    }),
+    [props.cornerRadius, isLightMode],
+  );
+
+  const innerCircleIndicatorProps = React.useMemo(
+    () => ({
+      height: props.cornerRadius * 2,
```

Comment:
This seems like it's not going to be handled well in the circular case. Probably should cap it to something reasonable?